pkg/info/types.go

```diff
@@ -15,10 +15,13 @@ const (
 	Java   Language = "java"
 )
 
+const MinimumClientVersion = ">= 1.2.0-0"
```

Comment:
let's create a `release.md` file and document that this field should be updated if there is a change?